news/gh-10795-conda-ci

```diff
@@ -0,0 +1,23 @@
+### Enhancements
+
+* Build Docker images periodically on GitHub Actions for the continuous
+  integration testing on Linux, storing them on GitHub Packages's registry
+  for reduced latency and cost when using Docker Hub.
+
+* Simplify the Linux GitHub actions workflows by combining used shell scripts.
```

Comment:
Thinking we should create a new "developer" section to clearly separate user-facing enhancements (which historically has been what this section was for) and developer-/contributor-facing changes.

Reply:
Good idea, I can go through the news items and move them into a developer section.